### DISCUSSION TITLE
Crashlytics and RN 0.68.2

### DISCUSSION THREAD

**Discussion by johnsoncwb** [invertase/react-native-firebase#6367] [Q&A]
Hello guys, i'm trying to install some dependencies in fresh expo bare workflow (RN 0.68.2). when i install ```@react-native-firebase/app``` and  ```@react-native-firebase/analytics``` everything works fine (v15.1.0) but when i install ```@react-native-firebase/crashlytics```  my android app just crash on open (on iOS with use_frameworks! it runs perfectly) but i don't find a way to run it on android. any idea or someone is using the v15.1.0 on android with RN0.68.2?

**Comment by mikehardy** [comment]
Any report of "my app is crashing" should have the crash trace from `adb logcat` every single time - we're just speculating until we see that and maybe (hopefully?) it's something easy that is obvious when you see the crash trace.

I haven't see anything on android that makes me think android has a problem. Right now the real difficulties are mostly around iOS and the need for `use_frameworks`, so an android issue is surprising. You'll note that I have a fully automated demo working here  - with crashlytics - https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh
